.github/workflows/release_update_readme.yaml

```diff
@@ -0,0 +1,68 @@
+---
+name: update-readme-on-release
+on:
+  release:
+    types: [published]
+permissions:
+  contents: read
+jobs:
+  update-readme:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+    - uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3  # v3.5.0
+      with:
+        ref: main
+    - name: Import GPG key
+      shell: bash
+      env:
+        BUILDBOT_GPG_KEY_B64: ${{ secrets.BUILDBOT_GPG_KEY_B64 }}
+        BUILDBOT_GPG_KEY_ID: ${{ secrets.BUILDBOT_GPG_KEY_ID }}
+      run: |
+        echo "${BUILDBOT_GPG_KEY_B64}" | base64 --decode | gpg --no-tty --batch --import
+        git config --global user.signingkey "${BUILDBOT_GPG_KEY_ID}"
+        git config --global commit.gpgsign true
+    - name: Import SSH key
+      shell: bash
+      env:
+        BUILDBOT_SSH_KEY_B64: ${{ secrets.BUILDBOT_SSH_KEY_B64 }}
```

Comment:
I think you can rely on the GH_TOKEN instead of needing a SSH_KEY if you clone over `https` instead of using the `git@` remote.
Feels marginally cleaner than doing the ssh key setup but this is fine too.

Reply:
I spent like 20min trying to get it to work with `https` but git was pushing as user `github-actions[bot]` instead of as `pixie-io-buildbot` and I tried a variety of things none of which worked, so just going to leave it with the SSH_KEY